### PR TITLE
Eventutils fixes

### PR DESCRIPTION
**Pull request by yipingbenjaminliu**

### CODE REVIEW

File: ap/accounts/models.py [hunk 1]
```diff
@@ -343,7 +343,17 @@ def __unicode__(self):
 
   # events in list of weeks
   def events_in_week_list(self, weeks):
-    schedules = self.active_schedules
+    active_sch_ids = []
```
Comment (iamsitting) [inline ap/accounts/models.py]: This var is not used.

File: ap/accounts/models.py [hunk 2]
```diff
@@ -343,7 +343,17 @@ def __unicode__(self):
...
+    for sch in self.active_schedules:
+      in_week = False
+      for wk in weeks:
+        if sch.active_in_week(wk):
+          in_week = True
+          break
+
+      if not in_week:
+        schedules = schedules.exclude(id=sch.id)
```
Comment (iamsitting) [inline ap/accounts/models.py]: Where does schedules come from? Doesn't seem defined.

File: ap/accounts/models.py [hunk 3]
```diff
@@ -352,10 +362,10 @@ def events_in_week_list(self, weeks):
     # return all the calculated, composite, priority/conflict resolved list of events
     return EventUtils.export_event_list_from_table(w_tb)
 
-  # TODO, work out case for users with two rolls for the same event and date
-  # currently just randomly grabs as seen with the rolls query
   def get_attendance_record(self):
```
Comment (iamsitting) [inline ap/accounts/models.py]: This needs to be reconciled with #1207

File: ap/accounts/models.py [hunk 4]
```diff
@@ -422,6 +432,17 @@ def events_in_date_range(self, start, end, listOfSchedules=[]):
     c_term = Term.current_term()
     start_week = c_term.term_week_of_date(start)
     end_week = c_term.term_week_of_date(end)
+
+    # only use the schedules that are active in those week ranges    
+    for sch in schedules:
```
Comment (iamsitting) [inline ap/accounts/models.py]: Can we make this a resuable method?

File: ap/aputils/eventutils.py [hunk 5]
```diff
@@ -99,13 +101,13 @@ def collapse_priority_event_trainee_table(weeks, schedules, t_set):
     return w_tb
 
   @staticmethod
-  def export_typed_ordered_roll_list(w_tb, type):
+  def export_typed_ordered_roll_list(w_tb, type=None, monitor=None):
```
Comment (iamsitting) [inline ap/aputils/eventutils.py]: the monitor arg is not used in the project. Will it be used?

**Comment by yipingbenjaminliu** [comment]
temp closing this, this factors into a larger issue with schedules.

**Pull request closed without merging**